pkg/ingestor/parser/spdx/parse_spdx.go

```diff
@@ -183,6 +183,8 @@ func (s *spdxParser) GetPredicates(ctx context.Context) *assembler.IngestPredica
 
 	toplevel := s.getPackageElement("DOCUMENT")
 	// adding top level package edge manually for all depends on package
+	// TODO: This is not based on the relationship so that can be inaccurate (can capture both direct and in-direct)...Remove this and be done below by the s.spdxDoc.Relationships?
+	// see https://spdx.github.io/spdx-ntia-sbom-howto/ section 4.1.2. Additional subdependencies
 	if toplevel != nil {
 		preds.IsDependency = append(preds.IsDependency, common.CreateTopLevelIsDeps(toplevel[0], s.packagePackages, s.filePackages, "top-level package GUAC heuristic connecting to each file/package")...)
```

Comment:
Based on the direct/indirect information we have, appending all packages to the top level seems like a false statement...

Reply:
changed to `UKNOWN` for dependency type...